CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 ## [25.0.7] - 2020-10-09
 
 ### Fixed
-
+- SC-6735 administration remove consent triggers import hash generation
```

Comment:
As hotfix you must add new package version number. in package and package-lock and to changelog

Reply:
I see @LukasSielski has already added this comment. Please do this as change request to block merges.

Reply:
Done, Thanks @CeEv